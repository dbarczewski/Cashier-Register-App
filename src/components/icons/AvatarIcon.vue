<template>
  <div
    class="rounded-full w-8 h-8 bg-green-800 flex justify-center items-center"
  >
    <span>VK</span>
  </div>
</template>

<script lang="ts">
import { defineComponent } from "vue";

export default defineComponent({
  name: "AvatarIcon",
});
</script>
