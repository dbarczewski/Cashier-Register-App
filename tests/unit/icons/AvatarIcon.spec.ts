
import { shallowMount } from "@vue/test-utils";
import AvatarIcon from "@/components/icons/AvatarIcon.vue";

describe("icons/AvatarIcon", () => {
  it("renders props.msg when passed", () => {
    const wrapper = shallowMount(AvatarIcon);
    expect(wrapper.text()).toMatch("AvatarIcon");
  });
});

