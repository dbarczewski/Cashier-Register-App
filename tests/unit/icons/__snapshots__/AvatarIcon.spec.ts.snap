// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icons/AvatarIcon renders component correctly 1`] = `
<div
  class="rounded-full w-8 h-8 bg-green-800 flex justify-center items-center"
>
  <span>
    VK
  </span>
</div>
`;
